File: ditto-chat-server/src/main/java/com/ditto_chat/ditto_chat_server/dtos/AccountImageForm.java
package com.ditto_chat.ditto_chat_server.dtos;

public class AccountImageForm {
    private String accountImageFileS3ObjectKey;

    public AccountImageForm(String accountImageFileS3ObjectKey) {
        this.accountImageFileS3ObjectKey = accountImageFileS3ObjectKey;
    }

    public String getAccountImageFileS3ObjectKey() {
        return accountImageFileS3ObjectKey;
    }
}

File: ditto-chat-client/src/app/classes/AccountImageForm.ts
export default class AccountImageForm {
    private accountImageFileS3ObjectKey: string;

    public constructor(accountImageFileS3ObjectKey: string) {
        this.accountImageFileS3ObjectKey = accountImageFileS3ObjectKey;
    }

    public getAccountImageFileS3ObjectKey(): string {
        return this.accountImageFileS3ObjectKey;
    }
}
